test/test_server.py

```diff
@@ -168,6 +170,21 @@ def test_error_from_url(self):
         assert 'entities' in data['results']
         assert len(data['results']['entities']) == 1
         assert data['results']['entities'][0]['text'] == 'ClickUp'
+
+    def test_escape_sequences_from_url(self):
+        url= "https://web.archive.org/web/20241009131438/https://musikderzeit.de/"
+        response = self._client.post('/entities/from-url', data=dict(url=url, language='de', title=1))
+        
+        data = response.json()
+        assert 'results' in datax
+        json_string = json.dumps(data['results'])
```

Comment:
After tinkering with this a bit, the reason I found the escape characters bc the chunk of code you gave me to get entities from url uses json.dumps(). The escape characters only appear in the output this way. When extracting text from response.json(), the text was already decoded into the characters being 'ü' instead of '\u00fc. So, the find escape characters test is only necessary depending on the function used for the user's output.